### ISSUE TITLE
cube CMakeLists.txt could get glslangvalidator from VulkanSDK rather than install from github

### ISSUE THREAD

**Issue by water-chika** [KhronosGroup/Vulkan-Tools#858]
VulkanSDK contains glslangvalidator. So cube could get glslangvalidator from VulkanSDK rather than installing glslangvalidator from github.

**Comment by charles-lunarg** [comment]
So unbeknownst to you, there is a bigger issue with getting glslang from github, namely that the version gotten was rather old. On windows, it would require the MSVC 2013 CRT library which caused cryptic linker errors if they were missing. 

Originally, this repo *built* glslang. Since that is a massive dependency, a while ago we moved to just downloading the built binaries directly. This now is felt to be even more code than necessary, as we can simply check in the SPIR-V code and use that. 

I have created a PR to do just that #859

It removes the 'fetch_glslangValidator.py' script, and instead finds glslang through CMake's `find_program` functionality, with some hints to use the VULKAN_SDK if its present. But this is put behind a build option, since the SPIR-V files are checked in, glslang is not a build requirement. It is only needed if a user wishes to modify the source .vert or .frag files and recompile the shaders themselves.
